src/redux/slice.js

```diff
@@ -0,0 +1,228 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import {
+  fetchRegions,
+  fetchCategories,
+  fetchRestaurants,
+  fetchRestaurant,
+  postLogin,
+  postReview,
+} from '../services/api';
+
+import { saveItem } from '../services/storage';
+
+import { equal } from '../Utils/utils';
+
+const initialReviewFields = {
+  score: '',
+  description: '',
+};
+
+const { actions, reducer } = createSlice({
+  name: 'application',
+  initialState: {
+    regions: [],
+    categories: [],
+    restaurants: [],
+    restaurant: null,
+    selectedRegion: null,
+    selectedCategory: null,
+    loginFields: {
+      email: '',
+      password: '',
+    },
+    accessToken: '',
+    reviewFields: {
+      ...initialReviewFields,
+    },
+  },
+  reducers: {
+    setRegions(state, { payload: regions }) {
+      return {
+        ...state,
+        regions,
+      };
+    },
+
+    setCategories(state, { payload: categories }) {
+      return {
+        ...state,
+        categories,
+      };
+    },
+
+    setRestaurants(state, { payload: restaurants }) {
+      return {
+        ...state,
+        restaurants,
+      };
+    },
```

Comment:
```suggestion
    setRestaurants: (state, { payload: restaurants }) => ({ ...state, restaurants }),
```

한 줄로 쓸 수 있어서 저는 이렇게 쓰는 편인데 어떤 차이가 있을까요?

Reply:
줄이 너무 길어지지 않으면 한 줄로 쓰는 편이 보기에 더 좋아 보이네요.